app/web/features/profile/locales/en.json

```diff
@@ -127,37 +128,64 @@
-    "rating_how": "How should I rate my experience?",
+    "rating_how": "How was your overall experience with {{name}}?",
     "rating_question": "How would you rate your overall experience with {{name}}?",
     "reference_form_heading_friend": "You met with {{name}}",
     "reference_form_heading_hosted": "You hosted {{name}}",
     "reference_form_heading_surfed": "You surfed with {{name}}",
+    "reference_form_heading_did_stay_surfed": "Did you stay with {{name}}?",
+    "reference_form_heading_did_stay_hosted": "Did you host {{name}}?",
     "writing_for_text": "You are writing a reference for the following person:",
-    "reference_submit_heading": "Thank you for leaving a reference!",
+    "reason_didnt_meetup": "Why didn't you meet up? (Optional)",
+    "reference_submit_heading": "Please review your reference",
     "reference_success": "Successfully wrote the reference!",
     "reference_type_not_available": "This reference type is not available for this user.",
     "required": "This step is required.",
-    "safety_priority": "Your safety is our priority. It is important that you remain comfortable when interacting with others within the Couchers community. Let us know how you felt regarding this person's behavior.",
-    "text_explanation": "Leave a note for your fellow Coucher's Guestbook –– say thank you, or let others know if you enjoyed your time with them.",
-    "thank_you_message": "We appreciate you taking the time to help us uphold our community values. Please look over your reference before submitting it.",
+    "safety_priority": "It is important that you remain comfortable when interacting with others within the Couchers community. Let us know how you felt regarding this person's behavior.",
+    "text_explanation": "Leave a reference and let people know how your experience was with this person. This will help others know what to expect with this person in the future.",
+    "text_explanation_hosted_surfed": "The reference you type below will be shared with {{name}} after they write a reference for you, or after 14 days. It will also be visible for the rest of the Couchers.org community to see.",
+    "text_label": "Let others know about your experience by sharing the context and any relevant details here.",
+    "thank_you_message": "We appreciate you taking the time to write a thoughtful reference which will help others in the future. Below is a summary which you can review before submitting.",
+    "thank_you_header": "Thank you for leaving a reference!",
+    "thank_you_references_available": "When hosting or surfing, references become visible when both people have submitted their references, or after 2 weeks, whichever comes sooner.",
+    "thank_you_support": "Thanks for your support!",
```

Comment:
Most of all, thanks for being on Couchers and a part of our community!

:)

feel free to ignore